rickshaw-run

```diff
@@ -34,7 +34,7 @@ use toolbox::logging;
 use toolbox::run;
 use toolbox::jsonsettings;
 
-$toolbox::logging::debug = 0;
+$toolbox::logging::debug = 1;
```

Comment:
We don't want to merge this change.  I assume this was for debugging purposes...

Reply:
yes